storybook/pages/DAppConfirmDisconnectPopupPage.qml

```diff
@@ -0,0 +1,38 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import Qt.labs.settings 1.0
+
+import Models 1.0
+import Storybook 1.0
+
+import shared.popups.walletconnect 1.0
+
+
+Item {
+    id: root
+
+    DAppConfirmDisconnectPopup {
+        anchors.centerIn: parent
+        visible: true
+
+        dappIcon: settings.dappIcon
+        dappUrl: settings.dappUrl
+        dappName: settings.dappName
+
+        destroyOnClose: false
+
+    }
+
+    Settings {
```

Comment:
Why do you need these `Settings`? This should be some user-editable control panel, like in other SB pages imo